deploy/awstools/awstools.py

```diff
@@ -184,14 +229,20 @@ def launch_instances(instancetype, count, instancemarket, spotinterruptionbehavi
 
     # starting with the first subnet, keep trying until you get the instances you need
     startsubnet = 0
-    instances = []
-    while len(instances) < count:
-        if not (startsubnet < len(subnets)):
-            rootLogger.critical("we tried all subnets, but there was insufficient capacity to launch your instances")
-            rootLogger.critical("""only the following {} instances were launched""".format(len(instances)))
-            rootLogger.critical(instances)
-            return
 
+    if tags and not additive:
+        instances = instances_sorted_by_avail_ip(get_instances_by_tag_type(tags, instancetype))
+    else:
+        instances = []
+
+    if len(instances):
+        rootLogger.info("Already have {} of {} {} instances.".format(len(instances), count, instancetype))
```

Comment:
It seems to me this would underreport the number of instances in the additive mode.

Reply:
In the additive mode, it won't print because instances will be empty at this point.